diagnostic.md

```diff
@@ -73,13 +84,21 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request POST http://www.example.com/donuts/
+{
+"name": "French Cruller",
+  "price": "$0.99"
+}
 ```
 
 Write an AJAX request to create a single donut on the server using JSON.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = $.ajax({
+    url: http://www.example.com/donuts/,
+    method: 'POST',
+    data: { "name": "French Cruller", "price": "$0.99" },
```

Comment:
Almost, you need the object name of the data you're sending. In this case, `Donut`
```js
let createDonut = function () {
  return $.ajax({
    url: 'http://example.com/donuts',
    method: 'POST',
    data: {
      "donut" {
        "name": "French Cruller",
        "price": "$0.99"
      }
    }
  });
};
```